test/integration/puppetlabs/services/jruby/jruby_puppet_pool_int_test.clj

```diff
@@ -284,7 +267,7 @@
            (tk-app/stop app)))))))
 
 (deftest ^:integration test-503-when-jruby-is-first-to-shutdown
-  (testing "During a shutdown requests result in 503 http responses"
+  (testing "After a shutdown, requests result in 503 http responses"
```

Comment:
I had to go look at those again for a while, but I think you're right. I can add some detail, because it was really confusing